src/services/recipient.js

```diff
@@ -511,6 +512,13 @@ export async function getGoalsByActivityRecipient(
     ...filters
   },
 ) {
+  // Get the GoalTemplateFieldPrompts where title is 'FEI root cause'.
+  const feiRootCauseFieldPrompt = await GoalTemplateFieldPrompt.findOne({
+    where: {
+      title: 'FEI root cause',
+    },
+  });
```

Comment:
since you only use the column goalTemplateId, please filter result set to only that column